src/SkipOnAll.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Validator;
+
+final class SkipOnAll
```

Comment:
Was it meant to be named `DoNotSkip` or `NeverSkip`?

Reply:
Yes, you are right since it returns `false`. I choose `SkipNever` for better grouping. Should these classes be placed to a separate folder?

Reply:
Logically yes. I think si.

Reply:
:heavy_check_mark: